tests/affine-cloud-copilot/e2e/utils/settings-panel-utils.ts

```
@@ -207,4 +207,63 @@ export class SettingsPanelUtils {
       await searcher.getByTestId('doc-selector-confirm-button').click();
     }
   }
+
+  private static async waitForEmbeddingStatus(
+    page: Page,
+    timeout: number,
+    status = 'synced'
+  ) {
+    await expect(async () => {
+      await this.openSettingsPanel(page);
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)